dell/precision/5560/default.nix

```diff
@@ -0,0 +1,40 @@
+{
+  config,
+  lib,
+  ...
+}: {
+  imports = [
+    ../../../common/pc/laptop
+    ../../../common/pc/laptop/ssd
+    ../../../common/cpu/intel
+    ../../../common/gpu/intel/tiger-lake
+    ../../../common/gpu/nvidia/prime.nix
+    ../../../common/gpu/nvidia/turing
+  ];
+
+  hardware.enableRedistributableFirmware = lib.mkDefault true;
+
+  boot = {
+    blacklistedKernelModules = ["nouveau"];
+    kernelModules = ["kvm-intel"];
+    kernelParams = ["i915.modeset=1"];
+  };
+
+  hardware = {
+    graphics = {
+      enable = lib.mkDefault true;
+      enable32Bit = lib.mkDefault true;
+    };
+    nvidia = {
+      package = config.boot.kernelPackages.nvidiaPackages.beta;
```

Comment:
can you put a version number here, so that we can remove this once the stable nvidia package works as well?

Reply:
removed it entirely